src/views/SignTransactionLedger.vue

```diff
@@ -153,8 +166,22 @@ export default class SignTransactionLedger extends Vue {
                 label: this.rpcState.origin.split('://')[1],
                 image: checkoutRequest.shopLogoUrl,
             };
+        } else if (this.request.kind === RequestType.CASHLINK) {
+            // coming from cashlink create
+            if (!this.cashlink) {
+                // happens if user reloads the page
+                this._back();
+                return;
+            }
+            senderUserFriendlyAddress = this.$store.state.activeUserFriendlyAddress;
+
+            this.recipientDetails = {
+                address: this.cashlink!.address.toUserFriendlyAddress(),
+                label: 'New Cashlink',
+                isCashlink: true,
+            };
         } else {
-            this.$rpc.reject(new Error('Must be invoked via sign-transaction or checkout requests.'));
+            this.$rpc.reject(new Error('Must be invoked via sign-transaction, checkout or cashlink requests.'));
```

Comment:
Can you add to this error message _what_ must be invoked? Like maybe 'Ledger Transaction Signing must be invoked...'? That would be a more helpful error message by itself without having to look at the stack trace.